upload-riffraff.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Didn't know about `before_cache`, I like what you do there, will change

I'm using a build matrix to speed up the build, `continuous-integrations.sh` gets called twice. If I upload from `continuous-integrations` it uploads twice, and if I upload only during scala test it uploads even when JS tests fail.

That's why I have two separate steps and `upload-riffraff.sh` is only called on `after_success`